app/screens/ExploreScreen/ExploreScreen.tsx

```diff
@@ -109,7 +105,7 @@ export const ExploreScreen: FC<TabScreenProps<"Explore">> = () => {
           stickySectionHeadersEnabled={false}
           sections={sections}
           renderSectionHeader={({ section: { title, data } }) =>
-            data.length > 0 ? (
+            data[0].length > 0 ? (
```

Comment:
I'm guessing we're expecting more to be added to `data`?

Reply:
Thats correct. Based on the WebFlow structure, that is a possibility, but for now we're not using it. I wanted to keep it a possibility for the design team